test/chol.jl

```diff
@@ -5,18 +5,26 @@ using PDMats: chol_lower, chol_upper
     @testset "allocations" begin
         A = rand(100, 100)
         C = A'A
+        invC = inv(C)
         size_of_one_copy = sizeof(C)
         @assert size_of_one_copy > 100  # ensure the matrix is large enough that few-byte allocations don't matter
 
         @test chol_lower(C) ≈ chol_upper(C)'
         @test (@allocated chol_lower(C)) < 1.05 * size_of_one_copy  # allow 5% overhead
         @test (@allocated chol_upper(C)) < 1.05 * size_of_one_copy
 
+        X = randn(d, 10)
```

Comment:
What is this one?

Reply:
`d` should be 100 and used above as well. It's a remaining (final?) issue from the merge commit https://github.com/JuliaStats/PDMats.jl/pull/168/commits/afaf6999a7c6d6a55042b3b6a5342768faf75600 that broke tests a bit.